docs/how-to/installation/install-on-kubernetes.txt

```diff
@@ -0,0 +1,166 @@
+
+
+Prerequisites
+-------------
+
+Before installing Determined on a Kubernetes cluster, please ensure that the
+following prerequisites are satisfied:
+
+* The Kubernetes cluster should be running Kubernetes >= 1.15, with GPU support enabled. :ref:`Instructions for setting up a GKE cluster <setup-gke-cluster>`.
+* You should have access to the cluster via `kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_.
+* `Helm <https://helm.sh/docs/helm/helm_install/>`_ should be installed.
+* You should have a local copy of the:download:`Determined Helm Chart </helm/determined-helm-chart.tgz>`.
+
+
+What Gets Installed
+-------------------
+
+When the Determined Helm chart is installed, the following entities will be created:
+
+#. Deployment of the Determined master.
+#. ConfigMap containing configurations for the Determined master.
+#. LoadBalancer service to make the Determined master accessible.
+#. ServiceAcccount which will be used by the Determined master.
+#. Deployment of a Postgres database. Later in this guide, we demonstrate how an external database can be used instead.
+#. PersistentVolumeClaim for the Postgres database. Omitted if using an external database.
+#. Service to allow the Determined master to communicate with the Postgres database. Omitted if using an external database.
+
+
+.. _configure-determined-kubernetes-version:
+
+Configuring Which Version of Determined to Install
+--------------------------------------------------
+
+To configure which version of Determined users need to modify ``appVersion`` in ``Chart.yaml``. Users can specify
+a release version (e.g., ``0.13.0``) or specify any commit hash from the `upstream Determined repo <https://github.com/determined-ai/determined>`_
+(e.g., ``b13461ed06f2fad339e179af8028d4575db71a81``). Users are encouraged to use a released version.
+
+.. note::
+
+    If using a non-release branch of the Determined repository, ``appVersion`` is going to be set to ``X.Y.Z.dev0``.
+    This is not an official release version and deploying this version will result in an ``ImagePullBackOff`` error.
+    Users should remove ``.dev0`` to get the released version, or they can specify a specific commit hash
+    instead.
+
+
+Specifying the Number of GPUs Per Node
+--------------------------------------
+
+Users are required to specify the number of GPUs on each node (for GPU-enabled nodes only). This is done
+by setting ``slotsPerNode`` in ``values.yaml``. Determined uses this information when scheduling multi-GPU tasks.
+Each multi-GPU (distributed training) task will be scheduled as a set of ``slotsPerTask / slotsPerNode`` separate pods,
+with each pod assigned to ``slotsPerNode`` GPUs. Tasks with sizes that are not divisible by ``slotsPerNode`` are never scheduled.
+If you have a cluster of different size nodes (e.g., some nodes with 4 GPUs and
+other nodes with 8 GPUs), set ``slotsPerNode`` to the smallest common denominator.
```

Comment:
Can we elaborate on the ramifications of having different node types in the same cluster? i.e., does that mean only the smallest common denominator of GPUs will be used?

Reply:
Reworded and added a sentence to try and clarify this.